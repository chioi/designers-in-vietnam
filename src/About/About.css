.about-list {
  list-style: initial;
}

.about-container {
  padding: 0;
  max-width: 574px;
  margin: auto;
}

@media screen and (min-width: 1113px){
  .about-container {
    padding: 0 335px;
    max-width: 574px;
    margin: auto;
  }
}
