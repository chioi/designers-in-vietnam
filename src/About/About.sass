@import "../style/variables"
@import "../style/mixins/space"
@include padding-placeholders($space)

.about-list
  list-style: initial
  line-height: 26px

.about-container
  @extend %padding-top-big
  max-width: 574px
  margin: auto

@media screen and (min-width: 1114px)
  .about-container
    padding: 0 335px
