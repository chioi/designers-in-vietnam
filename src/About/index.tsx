import { Link, RouteComponentProps } from "@reach/router";
import React, { FC } from "react";

const About: FC<RouteComponentProps> = () => (
  <div style={{padding: "0 335px"}}>
    <section>
      <h3 className="designer-name">What?</h3>
      <p>
        A Highly curated directory of professionals in the field of Design.
        Bringing together the Vietnam design community and making the design
        discipline visible for everyone else.
      </p>
    </section>
    <section>
      <h3 className="designer-name">Why?</h3>
      <p>Design is good, is our job to make it great!</p>
      <ul>
        <li>Bring designers together.</li>
        <li>Grow and learn together and from each other.</li>
        <li>Helping non-designers understand the value of design.</li>
        <li>Helping businesses see the full value of design.</li>
        <li>Making design less misunderstood.</li>
        <li>Generate a conversation about design beyond skills.</li>
        <li>Make design great again.</li>
      </ul>
    </section>
    <section style={{ display: "flex" }}>
      <div>
        <h3 className="designer-name">Are you a designer?</h3>
        <p>Request to be added</p>
        <p>* What kind of designers? Any kind!</p>
        <Link to={process.env.REACT_APP_FORM_URL}>
          <span className="margin-bottom-big">ðŸ›µ</span>
          <span>Join us</span>
        </Link>
      </div>
      <div>
        <h3 className="designer-name">Feedback or ideas?</h3>
        <p>This is just the beginning!</p>
        <p>Tell us what DOV can become.</p>
        <Link to={process.env.REACT_APP_FORM_URL}>
          <span className="margin-bottom-big">ðŸŒ±</span>
          <span>Leave feedback</span>
        </Link>
      </div>
    </section>
    <section>
      <h3 className="designer-name">The team</h3>
      <p>
        We are a couple of designers and developers from{" "}
        <span>Wizeline Vietnam</span> who got a gut feeling and wanted to do
        something about it!
      </p>
    </section>
    <section>
      <h3 className="designer-name">What's next?</h3>
      <ul>
        <li>Get the designer list up and running!</li>
        <li>Improve visual design, experience, and identity.</li>
        <li>Add events, resources, etc...</li>
      </ul>
    </section>
  </div>
);

export default About;
