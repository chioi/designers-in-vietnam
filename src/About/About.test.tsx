import React from "react";
import {create, ReactTestRenderer} from "react-test-renderer";
import About from "../About";

// TODO: Suppress the linter warnings
jest.mock("@reach/router", () => ({
  Link: ({ children, to }) => <a href={to}>{children}</a>,
}));

describe('gets created', () => {
  let component: ReactTestRenderer;

  beforeAll(() => {
    component = create(<About path="/about" />);
  });

  it('matches the snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
