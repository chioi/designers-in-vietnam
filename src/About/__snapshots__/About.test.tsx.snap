// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets created matches the snapshot 1`] = `
<div
  style={
    Object {
      "padding": "0 335px",
    }
  }
>
  <section>
    <h3
      className="designer-name"
    >
      What?
    </h3>
    <p>
      A Highly curated directory of professionals in the field of Design. Bringing together the Vietnam design community and making the design discipline visible for everyone else.
    </p>
  </section>
  <section>
    <h3
      className="designer-name"
    >
      Why?
    </h3>
    <p>
      Design is good, is our job to make it great!
    </p>
    <ul>
      <li>
        Bring designers together.
      </li>
      <li>
        Grow and learn together and from each other.
      </li>
      <li>
        Helping non-designers understand the value of design.
      </li>
      <li>
        Helping businesses see the full value of design.
      </li>
      <li>
        Making design less misunderstood.
      </li>
      <li>
        Generate a conversation about design beyond skills.
      </li>
      <li>
        Make design great again.
      </li>
    </ul>
  </section>
  <section
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <div>
      <h3
        className="designer-name"
      >
        Are you a designer?
      </h3>
      <p>
        Request to be added
      </p>
      <p>
        * What kind of designers? Any kind!
      </p>
      <a>
        <span
          className="margin-bottom-big"
        >
          ðŸ›µ
        </span>
        <span>
          Join us
        </span>
      </a>
    </div>
    <div>
      <h3
        className="designer-name"
      >
        Feedback or ideas?
      </h3>
      <p>
        This is just the beginning!
      </p>
      <p>
        Tell us what DOV can become.
      </p>
      <a>
        <span
          className="margin-bottom-big"
        >
          ðŸŒ±
        </span>
        <span>
          Leave feedback
        </span>
      </a>
    </div>
  </section>
  <section>
    <h3
      className="designer-name"
    >
      The team
    </h3>
    <p>
      We are a couple of designers and developers from
       
      <span>
        Wizeline Vietnam
      </span>
       who got a gut feeling and wanted to do something about it!
    </p>
  </section>
  <section>
    <h3
      className="designer-name"
    >
      What's next?
    </h3>
    <ul>
      <li>
        Get the designer list up and running!
      </li>
      <li>
        Improve visual design, experience, and identity.
      </li>
      <li>
        Add events, resources, etc...
      </li>
    </ul>
  </section>
</div>
`;
